src/Raven.Server/Documents/Patch/PatchDocumentCommand.cs

```diff
@@ -186,6 +186,8 @@ public override int Execute(DocumentsOperationContext context)
                             }
                         }
                     }
+
+                    _run.UpdatedDocumentCounterIds.Clear();
```

Comment:
Did this mean that we were updating _all_ the counters for all the previous documents on each operation?

Reply:
Yes